src/rollup/index.ts

```diff
@@ -1,204 +1,24 @@
-import {
-	getTimings,
-	initialiseTimers,
-	SerializedTimings,
-	timeEnd,
-	timeStart
-} from '../utils/timers';
+import { getTimings, initialiseTimers, timeEnd, timeStart } from '../utils/timers';
 import { basename } from '../utils/path';
 import { writeFile } from '../utils/fs';
 import { mapSequence } from '../utils/promise';
 import error from '../utils/error';
 import { SOURCEMAPPING_URL } from '../utils/sourceMappingURL';
 import mergeOptions, { GenericConfigObject } from '../utils/mergeOptions';
-import { ModuleJSON } from '../Module';
-import { RawSourceMap } from 'source-map';
-import Program from '../ast/nodes/Program';
-import { Node } from '../ast/nodes/shared/Node';
-import { WatcherOptions } from '../watch/index';
 import { Deprecation } from '../utils/deprecateOptions';
 import Graph from '../Graph';
-import { TransformContext } from '../utils/transform';
 import ensureArray from '../utils/ensureArray';
 import { SourceMap } from 'magic-string';
 import { createAddons } from '../utils/addons';
 import commondir from '../utils/commondir';
-
-export const VERSION = '<@VERSION@>';
```

Comment:
has this been removed on purpose?

Reply:
Well caught - yes I removed this one by mistake.